runtime/jcl/common/java_lang_invoke_MethodHandleNatives.cpp

```diff
@@ -1522,5 +1522,18 @@ Java_java_lang_invoke_MethodHandleNatives_registerNatives(JNIEnv *env, jclass cl
 	return;
 }
 
+jobject JNICALL
+Java_java_lang_invoke_MethodHandle_invokeExact(JNIEnv *env, jclass ignored, jobject handle, jobject args)
```

Comment:
Isn't this already present in the java_dyn_methodhandle.c file:
https://github.com/eclipse/openj9/blob/7fade16235106505d2d5a52a940d0fa0511ce76c/runtime/jcl/common/java_dyn_methodhandle.c#L792

Reply:
Can the two be commoned?

Reply:
Is it possible to build both all the time, and rely on the ifdefs for exclusion between ojdkmh and j9mh?